sql/search_derived/mobile_search_clients_daily_v1/query.sql

```diff
@@ -0,0 +1,168 @@
+    searches.value AS search_count,
+    UNIX_DATE(udf_iso8601_str_to_date(client_info.first_run_date)) AS profile_creation_date,
+    SAFE.DATE_DIFF(
+      udf_iso8601_str_to_date(ping_info.end_time),
+      udf_iso8601_str_to_date(client_info.first_run_date),
+      DAY
+    ) AS profile_age_in_days
+  FROM
+    org_mozilla_fenix_stable.metrics_v1
+  LEFT JOIN
+    fenix_client_locales ON client_id = client_info.client_id
+  CROSS JOIN
+    UNNEST(
+      -- Add a null search to pings that have no searches
+      IF(ARRAY_LENGTH(metrics.labeled_counter.metrics_search_count) = 0,
+         null_search(),
+         metrics.labeled_counter.metrics_search_count
+      )) AS searches
+),
+
+combined_search_clients AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    client_id,
+    normalized_search_key[SAFE_OFFSET(0)] AS engine,
+    normalized_search_key[SAFE_OFFSET(1)] AS source,
+    SUM(IF(ARRAY_LENGTH(normalized_search_key) = 0, 0, search_count)) AS search_count,
+    udf_mode_last(ARRAY_AGG(normalized_country_code)) AS country,
+    udf_mode_last(ARRAY_AGG(locale)) AS locale,
+    normalized_app_name AS app_name,
+    udf_mode_last(ARRAY_AGG(metadata.uri.app_version)) AS app_version,
+    udf_mode_last(ARRAY_AGG(normalized_channel)) AS channel,
+    udf_mode_last(ARRAY_AGG(normalized_os)) AS os,
+    udf_mode_last(ARRAY_AGG(normalized_os_version)) AS os_version,
+    udf_mode_last(ARRAY_AGG(default_search)) AS default_search_engine,
+    CAST(NULL AS STRING) AS default_search_engine_submission_url,
+    udf_mode_last(ARRAY_AGG(distribution_id)) AS distribution_id,
+    udf_mode_last(ARRAY_AGG(profile_date)) AS profile_creation_date,
+    udf_mode_last(ARRAY_AGG(profile_age_in_days)) AS profile_age_in_days,
+    udf_mode_last(ARRAY_AGG(sample_id)) AS sample_id
+  FROM
+    core_flattened_searches
+  GROUP BY
+    client_id,
+    submission_date,
+    engine,
+    source,
+    app_name
+  UNION ALL
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    client_info.client_id AS client_id,
+    normalized_search_key[SAFE_OFFSET(0)] AS engine,
+    normalized_search_key[SAFE_OFFSET(1)] AS source,
+    SUM(IF(ARRAY_LENGTH(normalized_search_key) = 0, 0, search_count)) AS search_count,
+    udf_mode_last(ARRAY_AGG(normalized_country_code)) AS country,
+    udf_mode_last(ARRAY_AGG(locale)) AS locale,
+    'Fenix' AS app_name,
+    udf_mode_last(ARRAY_AGG(client_info.app_display_version)) AS app_version,
+    udf_mode_last(ARRAY_AGG(normalized_channel)) AS channel,
+    udf_mode_last(ARRAY_AGG(normalized_os)) AS os,
+    udf_mode_last(ARRAY_AGG(client_info.android_sdk_version)) AS os_version,
+    udf_mode_last(ARRAY_AGG(metrics.string.search_default_engine_code)) AS default_search_engine,
+    udf_mode_last(ARRAY_AGG(metrics.string.search_default_engine_submission_url)) AS default_search_engine_submission_url,
+    CAST(NULL AS STRING) AS distribution_id,
```

Comment:
We _should_ have distribution information in Fenix. If we don't we should file an issue, but I'm not seeing it anywhere.

Reply:
There are open issues in Fenix for the distribution info so I assume it's not available yet.  By "should" do you mean that this should already be done or that this is something we want to have?

Reply:
By "should" I mean, if we don't we should. I thought it was in place but apparently not :)